dts/bindings/mtd/atmel,at45.yaml

```diff
@@ -62,3 +62,11 @@ properties:
       Time, in nanoseconds, needed by the chip to exit from the Deep Power-Down
       mode (or Ultra-Deep Power-Down mode when the "use-udpd" property is set)
       after the corresponding command is issued.
+
+  reset-gpios:
```

Comment:
Here you are using `reset-gpios` while in `struct spi_flash_at45_data` you are using `reset-gpios`.
Its all about one PIN, let's using singular.

Reply:
@nvlsianpu in yaml, singular not allowed:
`devicetree error: 'reset-gpio' in 'properties:' in zephyr/dts/bindings\mtd\atmel,at45.yaml is 'type: phandle-array', but its name does not end in -s. This is required since property names like '#pwm-cells' and 'pwm-names' get derived from 'pwms', for example.`